src/plugins/rhbz.c

```diff
@@ -682,7 +682,7 @@ int rhbz_attach_fd(struct abrt_xmlrpc *ax, const char *bug_id,
 
 //TODO: need to have a method of attaching huge files (IOW: 1Gb read isn't good).
 
-    char *data = xmalloc(size + 1);
```

Comment:
I can only assume that this was done in assumption that the last byte would somehow be zero to appease `strlen()` gods.